ipykernel/kernelbase.py

```diff
@@ -855,10 +856,26 @@ async def debug_request(self, stream, ident, parent):
         if inspect.isawaitable(reply_content):
             reply_content = await reply_content
         reply_content = json_clean(reply_content)
+        reply_content['hello'] = 'hello'
         reply_msg = self.session.send(stream, 'debug_reply', reply_content,
                                       parent, ident)
         self.log.debug("%s", reply_msg)
 
+    async def usage_request(self, stream, ident, parent):
+        content = parent['content']
+        reply_content = self.do_debug_request(content)
+        if inspect.isawaitable(reply_content):
+            reply_content = await reply_content
+        reply_content = json_clean(reply_content)
+        reply_content['cpu_percent'] = psutil.cpu_percent()
+        reply_content['virtual_memory'] = psutil.virtual_memory()
```

Comment:
It seems all of `virtual_memory` `virtual_memory_dict`, `virtual_memory_percent` and `available_memory_percentage` can be derived by just using `virtual_memory`.

Proposal - only send:
```
reply_content['virtual_memory'] = dict(psutil.virtual_memory()._asdict())
```

Reply:
Thx. Fixed with 889481b15413755f1cebfe433a78fead6559f67f